page.codeberg.terratactician_expandoria.game.metainfo.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>page.codeberg.terratactician_expandoria.game</id>
+
+	<name>TerraTactician Expandoria</name>
+	<icon type="stock">page.codeberg.terratactician_expandoria.game</icon>
+	<summary>Hexagon based logic game</summary>
+
+	<developer id="codeberg.org">
+		<name translate="no">TerraTactician-Expandoria Team</name>
+	</developer>
+
+	<screenshots>
+		<screenshot type="default">
+			<image>https://codeberg.org/terratactician_expandoria/artwork/raw/branch/main/screenshots/menu.png</image>
```

Comment:
screenshots should never point to the tip of a branch as it might break if things get change. Pin to a specific commit.

Reply:
Totally forgot about that when using the file browser :sweat_smile: